Demo/Sources/Recycling/GridViews/Markets/MarketsGridViewDemoView.swift

```diff
@@ -4,47 +4,229 @@
 
 import FinniversKit
 import DemoKit
+import UIKit
 
-class MarketDataSource: NSObject {
-    var models = Market.newMarkets
+class MarketsDemoView: UIView, Demoable {
+    var demoStack = UIStackView()
+    var finnDemo = FinnMarketsDemoView()
+    var finnDemo2 = FinnMarketsDemoView2()
+
```

Comment:
could you please run these files thorough a linter?
https://github.com/realm/SwiftLint
specifically, cd into the folder this file is in, and run swiftlint --fix
it will remove the warnings for the most of us which have linters installed 🙂

Reply:
tried running swiftlint --fix but got:  SourceKittenFramework/library_wrapper.swift:31: Fatal error: Loading sourcekitdInProc.framework/Versions/A/sourcekitdInProc failed
zsh: trace trap  swiftlint --fix

Reply:
😕